src/test/java/de/szut/lf8_project/Lf8ProjectApplicationTests.java

```diff
@@ -1,12 +1,15 @@
 package de.szut.lf8_project;
 
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+@Disabled
 @SpringBootTest
 class Lf8ProjectApplicationTests {
 
     @Test
+    // Dieser Test funktioniert (natürlich) nicht in der Pipeline, weil er eine lokale DB vorraussetzt
```

Comment:
Können mal testen, ob wir hier den Container nutzen könnten, wenn MS-2 im Master ist.

Reply:
ich bin ansonsten auch onboard diesen test komplett zu killen. der App Context wird in jedem integrationstest geprüft